### ISSUE TITLE
next.js/examples/with-mongodb-mongoose/  From fields return strings not array in database

### ISSUE THREAD

**Issue by lemarcgagnon** [vercel/next.js#29734]
### What example does this report relate to?

next.js/examples/with-mongodb-mongoose

### What version of Next.js are you using?

latest

### What version of Node.js are you using?

14

### What browser are you using?

chrome, firefox

### What operating system are you using?

linux

### How are you deploying your application?

locally

### Describe the Bug

the form component for the diet field and other array fields declared in the Mongoose Model should be like so ["item1","item2,"item3"] and not ["item1 item2 item3"].  Verification should be done before - either taking in consideration for the <n or ,  - and then the array should be saved in the MongoDB.

### Expected Behavior

["item1","item2,"item3"] and not ["item1 item2 item3"]

### To Reproduce

type any string in the diet field, it will make an array of one long item instead of an array of items.

**Comment by balazsorban44** [comment]
This issue has been automatically closed because it wasn't verified against next@canary. If you think it was closed by accident, please leave a comment. If you are running into a similar issue, please open a new issue with a reproduction. Thank you.
